### ISSUE TITLE
Editor becomes unresponsive when console is filled

### ISSUE THREAD

**Issue by sicher** [defold/editor2-issues#749]
If running a game that quickly fills the console with output, the editor becomes unusable. This happens easily if you, for instance, have lots of physics objects and start running out of resources.

STR:

To simulate, run the following for 1-2 seconds:

```
function update(self, dt)
	for i=1,500 do
		print("hello, this is a pretty long output to make sure that we fill the console properly. It's gonna choke!")
	end
end
```

**Comment by Malmer** [comment]
I thought they did a hack fix to this in #322. Maybe it didn't work?

**Comment by RagnarSvenssonKing** [comment]
Fixed!
